core/src/main/java/com/linecorp/armeria/common/DefaultFlagsProvider.java

```diff
@@ -70,6 +70,7 @@ final class DefaultFlagsProvider implements FlagsProvider {
     static final long DEFAULT_MAX_SERVER_CONNECTION_AGE_MILLIS = 0; // Disabled
     static final long DEFAULT_MAX_CLIENT_CONNECTION_AGE_MILLIS = 0; // Disabled
     static final long DEFAULT_SERVER_CONNECTION_DRAIN_DURATION_MICROS = 1000000;
+    static final long DEFAULT_CLIENT_CONNECTION_DRAIN_DURATION_MICROS = 0; // Disabled
```

Comment:
What are your thoughts on setting the value to 1000000, which is the same as the server-side duration? I believe introducing a slight delay would be more sensible.

Reply:
On second thought, let's leave it as is. 😉

Reply:
There were some test failures when I tried setting it on 1000000 😢

Reply:
I'm curious, what kind of test failures did you get?